cmd/jaeger/internal/extension/jaegerquery/config_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2024 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jaegerquery
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_Validate(t *testing.T) {
+	tests := []struct {
+		name        string
+		config      *Config
+		expectedErr error
+	}{
+		{
+			name:        "Empty config",
+			config:      &Config{},
+			expectedErr: errors.New("TraceStoragePrimary: non zero value required"),
+		},
+		{
+			name: "Non empty-config",
+			config: &Config{
+				TraceStoragePrimary: "some-storage",
+			},
+			expectedErr: errors.New("TraceStoragePrimary: non zero value required"),
```

Comment:
is it expected?

Reply:
It was not, thanks for pointing this out.